css/project.less

```diff
@@ -148,6 +152,32 @@
   &.loading .load-ui {
     display: block;
   }
+}
+
+.project-resizer-wrapper {
+  .autohide & {
+    display: none;
+  }
 
-  
-}
\ No newline at end of file
+  position: absolute;
+  height: 100%;
+  right: 0;
+  top: 0;
+
+  width: 16px;
+
+  .project-resizer {
+    &:extend(.project-resizer-wrapper);
```

Comment:
I know that it would be better to handle this with pseudo elements from the css point of view, but unfortunately we need to track the click on the bar and pseudo-elements do not allow this.
The outside wrapper here is just to give the sidebar resizer some delay before appearing, so that the cursor does not flicker when the user is moving the mouse between the editor area and the sidebar (cursor changes to `resize-col` when over resizer).

PS Just thought it would also be nice to add `pointer-events: none` there (will update pr)